### ISSUE TITLE
[html] hover for html entities

### ISSUE THREAD

**Issue by dolsem** [microsoft/vscode#107326]
When you hover over an html entity, it would be nice to show a preview of the character or short description of how it would look on the page, etc. Showing Unicode equivalent would be neat as well. E.g. _U+00027_ **`'`** for `&apos;`, _U+000A0_ **`non-breaking space`** for `&nbsp;`.

**Comment by aeschli** [comment]
If anyone is interested in giving this a try, pull requests at https://github.com/Microsoft/vscode-html-languageservice are welcome.
We already have code completion support for entities

**Comment by steve1998** [comment]
I would like to give this a try. I'm new to open source and since this feature is to be implemented on https://github.com/Microsoft/vscode-html-languageservice, do I follow the same guidelines as contributing to the core editor (this repo) or is there a different specific contributing guidelines for that repo and any general tips? (such as building, testing, etc.)

**Comment by aeschli** [comment]
@steve1998 I updated the README of the service: https://github.com/microsoft/vscode-html-languageservice#development

**Comment by steve1998** [comment]
@aeschli Thank you for the update on the README.md. I also saw that for every entity in the `htmlEntities.ts`, there is already a unicode value for each entity. My question is can I modify each of the values of these entities to include either the glyph value or a short description along with the unicode for each entity, or should I not touch this file at all?

**Comment by steve1998** [comment]
@aeschli Thank you for updating the README. I've now encountered a problem when trying to do `yarn link` on the `vscode/extensions/html-language-features/server` directory. It seems that there is already a link present on that folder called `html-language-features` and when I fire up the Code OSS, it also says HTML language server crashed and non of the autocomplete or hover features were working for html files. Is this a known behavior or am I going in the wrong direction?

Steps that I did:
1. Built `vscode` 
2. `yarn link --link-folder <my directory of vscode-html-languageservice` on `vscode/extensions/html-language-features/server` which says it can't be linked since `html-languageserver` is registered
3. Fire up code oss using `./scripts/code.bat` and says that HTML language server can't start, module missing

Here are the logs of the issues:
![problem1](https://user-images.githubusercontent.com/20303709/94598818-662a1e00-0244-11eb-985c-f1c16edd3ea9.png)
![problem2](https://user-images.githubusercontent.com/20303709/94598819-66c2b480-0244-11eb-875b-c7d1c27a932d.png)
Here, there's no code oss process that has the `html-language-features` path
![problem3](https://user-images.githubusercontent.com/20303709/94598822-66c2b480-0244-11eb-82d1-a5e6463c08d7.png)
